### PR TITLE
✨[Smartadserver ad extension] Implement support for Fast Fetch

### PR DESCRIPTION
**Pull request by smart-adserver**
The change implements support for Fast Fetch ad rendering of the Smartadserver ad network. This introduces an extension as well as removes Smartadserver from the vendors.

### CODE REVIEW

**Comment by alanorozco** [comment]
@ampproject/wg-monetization

**Comment by calebcordry** [comment]
Hi @krysztal-smart thanks for the contribution! For these fast-fetch implementations we still want to use the existing iframe rendering logic in `amp-a4a.js` so that we can ensure some of AMP's security, privacy and UX principles. This involves things like ensuring only AMPHTML ads are served into friendly iframes and not shifting content by collapsing the ad when it does not fill. 

I am happy to schedule a time to sync for a new implementation like this so that I can try guide you through the process. Let me know if you are interested.

File: extensions/amp-ad-network-smartadserver-impl/0.1/amp-ad-network-smartadserver-impl.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
+/* eslint-disable local/no-forbidden-terms */
```
Comment (powerivq) [inline extensions/amp-ad-network-smartadserver-impl/0.1/amp-ad-network-smartadserver-impl.js:1]: Can you find out why is this needed? I do not believe you can disable a check for an entire file (it will fail some other tests). It will be better to fix the underlying issue.

File: build-system/compile/bundles.config.extensions.json [hunk 2]
```diff
@@ -1286,4 +1291,4 @@
       "wrapper": "bento"
     }
   }
-]
+]
```
[the remark this answers is not in the document]
Comment (krysztal-smart) [reply build-system/compile/bundles.config.extensions.json]: Fixed

File: extensions/amp-ad-network-smartadserver-impl/0.1/amp-ad-network-smartadserver-impl.js (hunk 1 above)
Comment (krysztal-smart) [reply extensions/amp-ad-network-smartadserver-impl/0.1/amp-ad-network-smartadserver-impl.js]: Removed. It was needed just for debugging.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: extensions/amp-ad/amp-ad.md [hunk 3]
```diff
@@ -444,7 +444,6 @@ See [amp-ad rules](validator-amp-ad.protoascii) in the AMP validator specificati
 -   [Sklik](../../ads/vendors/sklik.md)
 -   [SSP](../../ads/vendors/ssp.md)
 -   [SlimCut Media](../../ads/vendors/slimcutmedia.md)
--   [Smart AdServer](../../ads/vendors/smartadserver.md)
```
Comment (powerivq) [inline extensions/amp-ad/amp-ad.md:447]: Might want to keep this but link to the moved file.

**Comment by smart-adserver** [comment]
@calebcordry, @powerivq, @mlapeyre3, I'm closing this messy pull request and opening a new, clean [one](https://github.com/ampproject/amphtml/pull/36991) from our new company fork.
